markdown/2017-10-16.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Weekly Digest
+show_social: true
+show_agenda: true
+show_sponsorship: true
+...
+
+---
+
+<center>
+<div style="position:relative;width:50%">
+    ![S'mores](https://i.imgur.com/04T4tgi.png "But is it a biscuit? This age-old question remains unanswered.")
+</div>
+</center>
+
+---
+
+Welcome back to another week of DoCSoc events. This week we bring you a
+**Netcraft Workshop** and our first Bar Night: the **Welcome Bar Night**. More
+details below!
+
+Last week we advertised an event run by one of our sponsors, J.P. Morgan.
+It's important to point out that *we did not run this event*. We apologise
+that J.P. Morgan did not communicate the location of the event as effectively
+as they could've perhaps done.
+
+Last week's annual Sponsors' Exhibition saw over 500 members meeting the 14
+sponsors present, which is an amazing turnout. Thank you to everyone who
+attended the event!
+
+---
+
+Our upcoming events:
+
+# Consumer Credit Checking for Fun & Profit
+
+<center>
+    ![Netcraft Workshop Header](https://i.imgur.com/jdaLsHV.png)
+</center>
+
+- **When:**  <font color="red">**TONIGHT**</font> 17.30-23.00, 16th October 2017
+- **Where:** RSM G01 & the Union Bar thereafter
+- **Link:** [Facebook](https://www.facebook.com/events/1866631386986389/)
+
+Netcraft is running a workshop where you can learn some of the opportunities
+that exist to make catastrophic mistakes when developing and maintaining a
+web application on the Internet.
+
+What sort of programming error can be that expensive? How much is it worth to
+the lawyers? And how much is a mistake like that worth to the hacker who
+finds it?
+
+Your challenge is that you are looking at a credit agency's web interface
+holding credit information on hundreds of millions of people.
+
+Pizza and drinks will be provided at the event. In order to give an accurate
+idea how many staff Netcraft should bring and how much pizza we should buy,
+please press 'Going' on the Facebook event if you intend to come.
+
+There will a free bar night at the Union Bar after the competition.
+
+# The Welcome Bar Night
+
+<center>
+    ![Welcome Bar Night Header](https://i.imgur.com/tMtZTiv.png)
+</center>
+
+- **When:** 19.00-00.00, 19th October 2017
+- **Where:** Metric
+- **Link:** [Facebook](https://www.facebook.com/events/857222977758027/)
+
+It's time for DoCSoc's first bar night of the year: The Welcome Bar Night!
+
+Come along for free drinks and to say hi to our new freshers or catch up with
+friends - it's sure to be a night you don't want to miss out on.
+
+Drinks tokens will be given upon entry for FREE to all DoCSoc members who
+acquired membership before 18.00 on the day of the event. Strictly 18+ only.
```

Comment:
worth reminding people that DoC, EIE, JMC are automatically members?

Reply:
Yes good idea

Reply:
Computing, JMC, EIE = DoC tho 😛

Reply:
EIE aren't technically DoC - the course is run solely by EEE Dept